image-loader/app/lib/ImageLoaderConfig.scala

```diff
@@ -10,6 +10,8 @@ class ImageLoaderConfig(resources: GridConfigResources) extends CommonConfig(res
   val imageBucket: String = string("s3.image.bucket")
 
   val thumbnailBucket: String = string("s3.thumb.bucket")
+  val quarantineBucket: String = stringDefault("s3.quarantine.bucket", "")
```

Comment:
`stringOpt` is a much better way of expressing a configuration option that might not be present. This will give you an Option[String].